### ISSUE TITLE
Edit downtimes

### ISSUE THREAD

**Issue by EmmaTinten** [Icinga/icingaweb2#4542]
## Describe the solution you'd like
As a user I want a possibility to modify my previously created downtime, e.g. unchecking the box "all services". At the moment there's only the way to delete and recreate a downtime.

**Comment by nilmerg** [comment]
> At the moment there's only the way to delete and recreate a downtime.

Yes, and that's the only way for the foreseeable future unless Icinga 2 doesn't implement this natively in its own API. (Luckily it will really remove the downtimes resulting from the `all_services` flag since v2.13. :tada: )

There's really a *very* low chance that editing is possible sometime. I'm also tempted to say never. Though, there **may** be a chance for this in [Icinga DB Web](https://github.com/Icinga/icingadb-web), but I can't promise anything.
